src/helpers/routes.ts

```diff
@@ -22,3 +22,20 @@ export const routes = {
     index: 'swap',
   },
 };
+
+// absolute routes are used when it is unknown from which page exactly will navigation/redirect happen
+// therefore the whole path must be constructed
+export const absoluteRoutes = {
```

Comment:
can't we combine routes and absolute routes into one object? we can use the absolute routes everywhere